src/geode/geometry/basic_objects/circle.cpp

```diff
@@ -67,6 +69,33 @@ namespace geode
     {
         return radius_;
     }
+    template < typename PlaneType >
+    BoundingBox3D GenericCircle< PlaneType >::bounding_box() const
+    {
+        const auto angle = [this]( Vector3D direction ) {
+            double dP = plane_.normal().dot( direction );
+            if( dP <= -1.0 )
+            {
+                return M_PI;
+            }
+            if( dP >= 1.0 )
+            {
+                return 0.;
+            }
```

Comment:
if direction and normal are normalized, is this possible / needed?

Reply:
it only works if they are normalized but it is the case here